### ISSUE TITLE
PPV Live - Extra information

### ISSUE THREAD

**Issue by jengajenga** [WWBN/AVideo#6089]
@DanielnetoDotCom 

Looking at the ppv live, there should be some place to add extra information that will be visible to the buyer. See the screenshot

![ppv-live](https://user-images.githubusercontent.com/65451390/146325055-f881082f-e42c-494b-bdd7-3ca00f6259d8.jpeg)

One should be able to 
1. give a description that will be seen under so that the buyer knows what he is buying
2. add imagery like a poster or something for marketing purposes
3. inform about time especially if the event is taking place in a different time zone

Basically, there needs to be more information that is useful to the buyer.

Same goes for the other ppv itself.

Thank you

**Comment by DanielnetoDotCom** [comment]
Hi, you are correct.

I just made some improvements on version 1.7

Now the description area support some basic HTML 

![chrome-capture](https://user-images.githubusercontent.com/18009293/146378321-12c327ce-f54f-4849-a9fa-21e6ae848ee5.gif)

and the description will appear on the buy page

![image](https://user-images.githubusercontent.com/18009293/146378297-442d61fe-77b8-489f-ab4f-18ec79d0844d.png)

**Comment by jengajenga** [comment]
Very nice!! Thank you very much.

Let me download it :)

**Comment by jengajenga** [comment]
Thanks for this update @DanielnetoDotCom 

Now one more question, when the client chooses the ppv-live plan, will the client put the html or is it still the admin?

Assuming there are 5 different clients all doing a ppv-live which are the same type for example a live rap music concert, but different venues and each client inside his own specific avideo account.

Is the new description in v1.7 for the admin or will each of the 5 client be able to put their own description?

**Comment by DanielnetoDotCom** [comment]
each user can choose the description for his PPV

admin has no control over it

**Comment by jengajenga** [comment]
Thank you Dan. I'm satisfied for now so I will close this.
